docs/how-to/create-initial-data-migrations.md

```diff
@@ -0,0 +1,111 @@
+
+These are the steps:
+
+   1. Export the data into JSON
+   1. Generate a python script from the JSON data
+
+### Prerequisites
+
+You must have Docker installed
+
+The initial data exists in a Google spreadsheet, such as [this one for People Depot][pd-data-spreadsheet]. There should be individual sheets named after the model names the data correspond to, such as `ProgramArea - Data`. The sheet name is useful for us to identify the model it corresponds to.
+
+The sheet should be formatted like so:
+
+- the first row contains the names of the field names in the model. The names must be exactly the same
+- rows 2 to n are the initial data for the model we want to turn into a script.
+
+## Convert the data into JSON
+
+   1. Export the data from the Google [spreadsheet][pd-data-spreadsheet]
+      1. Find the sheet in the document containing the data to export. Let's use the `ProgramArea - Data` data as our example. It's
+      1. Make sure that the first row (column names) is frozen. Otherwise, freeze it by selecting the first row in the sheet, then Menu > View > Freeze > Up to row 1
+      1. Export to JSON. Menu > Export JSON > Export JSON for this sheet
+   1. Save the JSON into a file
+      1. Select and copy all the JSON text
+      1. Paste it into a new file and save it as [ModelNameInPascalCase]_export.json under app/core/initial_data/
+      1. The Pascal case is important in the next step to generate a python script to insert the data. It must match the model's class name for this to work.
+
+      **Potential data issue**
+      There was a problem with the JSON exporter where it omitted the underscore in `occ_code`. It should be fixed now but it's good to pay attention to other column name problems and fix them in the [Google Apps script][apps-script] in the [spreadsheet][pd-data-spreadsheet]. You will find out when the data insertion fails if there's a problem.
+## Convert JSON into Python script
+
+   1. Start Docker
+   1. From project root, run
+
+      ```bash
+      ./scripts/buildrun.sh
+      ```
+
+
+   1. Go to the project root and run this command
+
+      ```bash
+      docker-compose exec web python scripts/convert.py core/initial_data/ProgramArea_export.json
+      ```
+
+   1. Check that there's a new file called `app/core/scripts/programarea_seed.py` and that it looks correct
+      1. You can run it to verify, but will need to remove that data if you care about restoring the database state
+         1. Run this command to run the script
+
+            ```bash
+            docker-compose exec web python manage.py runscript programarea_seed
+            ```
+
+         1. To remove the data, go into the database and delete all rows from `core_programarea`
+
+            ```bash
+            docker-compose exec web python manage.py dbshell
+
+            # now we have a shell to the db
+
+            # see if all the seed data got inserted
+            select count(*) from core_programarea;
+            # shows 22 rows
```

Comment:
We should be expecting 9 entries based on the current .json file

Reply:
fixed